src/tree-view/index.d.ts

```diff
@@ -14,6 +14,8 @@ export interface TreeLabelOverrides {
   IconContainer?: Override<{}>;
   ExpandIcon?: Override<{}>;
   CollapseIcon?: Override<{}>;
+  LeafIconContainer?: Override<{}>;
+  LeafIcon?: Override<{}>;
```

Comment:
`indentGuides` should be added into this file too

Reply:
Added.